text/game/gui.stringtable

```diff
@@ -3260,7 +3245,7 @@ Beschwörungen: Mächtige magische Effekte, die Sänger auslösen können, wenn
     </Entry>
     <Entry>
       <ID>637</ID>
-      <DefaultText>Durch den Erwerb eines Händlerstands kann ein Händler zufällige Güter in deiner Großen Halle verkaufen.</DefaultText>
+      <DefaultText>Durch den Erwerb eines Händlerstands kann ein Händler Gemischtwaren in deiner Großen Halle verkaufen.</DefaultText>
```

Comment:
Hm, vom ingame Standpunkt haben wir nicht das Konzept „Ritter“, aber Große (be)herrschende Familien. Daher wäre Herrenhalle besser.